docs/introduction/Car-registry-tutorial.rst

```diff
@@ -94,7 +97,7 @@ Special proposition and proof:
        ::
         CarSellOrderBox 
        
-       in two different ways: opened by the buyer and thus buy the car or opened by the seller and thus recall car sell order. Such proof creation requires two different API calls, but as a result, in both cases, we will have the same type of transaction with the same proof type. 
+       in two different ways: opened by the buyer and thus buys the car, or opened by the seller and thus recalls the car sell order. Such proof creation requires two different API calls, but as a result, in both cases, we will have the same type of transaction with the same proof type. 
```

Comment:
A SellOrderProposition is presented in two different ways: opened by the buyer (meaning they buy the car), or opened by the seller (Meaning the seller recalled the CarSellOrder). This proof creation requires two different API calls, but as a result in both cases, we will have the same type of transaction with the same proof type.